robotstatuschecker.py

```diff
@@ -54,17 +54,19 @@
 from robot.api import ExecutionResult, ResultVisitor
 from robot.utils import Matcher
 
+from __future__ import print_function
```

Comment:
Future imports should be before other imports and code. See https://docs.python.org/2/reference/simple_stmts.html#future